src/main/scala/com/cognite/sdk/scala/v1/propertyMap.scala

```diff
@@ -121,31 +133,40 @@ object DataModelInstanceQueryResponse {
 }
 
 final case class DataModelInstanceByExternalId(
-    items: Seq[CogniteExternalId],
-    model: DataModelIdentifier
+    model: DataModelIdentifier,
+    spaceExternalId: String,
+    items: Seq[CogniteExternalId]
 )
 
 final case class Edge(
+    override val spaceExternalId: Option[String],
     override val externalId: String,
     `type`: String,
-    startNode: String,
-    endNode: String,
+    startNode: DataModelIdentifier,
+    endNode: DataModelIdentifier,
     properties: Option[Map[String, DataModelProperty[_]]] = None
 ) extends PropertyMap(
       {
-        val propsToAdd: Map[String, DataModelProperty[_]] = Map(
-          "externalId" -> PropertyType.Text.Property(externalId),
-          "type" -> PropertyType.Text.Property(`type`),
-          "startNode" -> PropertyType.Text.Property(startNode),
-          "endNode" -> PropertyType.Text.Property(endNode)
-        )
+        val propsToAdd: Map[String, DataModelProperty[_]] =
+          spaceExternalId
```

Comment:
Same comment as above, I'd prefer to see this as a separate `val`. It's getting a bit messy here.

Reply:
they remove spaceExternalId at this level from DMS api so I remove this prop as well 😅